packages/near-membrane-base/src/environment.ts

```diff
@@ -202,7 +211,7 @@ export class VirtualEnvironment {
             const unsafeBlueDesc = (blueDescriptors as any)[key];
             // Avoid poisoning by only installing own properties from blueDescriptors
             // eslint-disable-next-line prefer-object-spread
-            const safeBlueDesc = ObjectAssign({ __proto__: null }, unsafeBlueDesc);
+            const safeBlueDesc = toSafeDescriptor(unsafeBlueDesc);
```

Comment:
toSafeDescriptor mutates the first argument, in this case this first argument is provided by remap, I don't recall who provides that descriptor, it is probably safe to mutate it, but I'm not sure from reading this part of the code.

Reply:
Ah, okay!